stories/src/API/repository/story.repository.ts

```diff
@@ -0,0 +1,31 @@
+export default class StoryRepository {
+  constructor(private _database: any) {}
+
+  async create(story: Object) {
+    return await this._database.Stories.create(story);
+  }
+
+  async update(storyId: string, data: any) {
+    const updatedStory = await this._database.Stories.update(...data, {
+      where: { id: storyId },
+    });
+    return updatedStory;
+  }
```

Comment:
Vê que em quase todos os métodos, você faz: `this._database.Stories.algumaCoisa`? Recomendaria guardar uma referência direto para stories, assim seus métodos fariam apenas: `this.stores.algumaCoisa.`